common/components/renderCbs/index.ts

```diff
@@ -0,0 +1 @@
+export * from './UnitConverter';
```

Comment:
This is a pretty interesting pattern, but took me a while to grok. Any reason you'd want something like this over inheritance & state, or HOCs?

Reply:
Anything Unit related is part of the BN refactor, it'll be a lot more clear there :) reason why I used a render callback is so it's easy to use within nested jsx, as you want to be using it as the direct parent of a input element